docs/5.community/5.framework-contribution.md

```diff
@@ -128,7 +128,7 @@ git checkout -b my-new-branch
 
 ### Set Up Documentation Website in Local Environment
 
-The Nuxt documentation is currently deployed within [nuxt/nuxt.com](https://github.com/nuxt/nuxt.com) as a layer.
+The Nuxt documentation is currently deployed within [nuxt/nuxt](https://github.com/nuxt/nuxt) as a layer.
```

Comment:
The `nuxt.com` project is still at https://github.com/nuxt/nuxt.com but not open-sourced yet.

Reply:
ok I see, my bad